Resources/views/Bootstrap/grid.html.twig

```diff
@@ -0,0 +1,11 @@
+{% extends '::base.html.twig' %}
+
+{% block header_title %}
+<h2>Grid</h2>
+{% endblock %}
+
+{% block main %}
+<div class="block-link">
+  <p> Meer info en voorbeelden op <a href="http://getbootstrap.com/css/#grid">http://getbootstrap.com/css/#grid</a></p>
```

Comment:
Is niet nuttig, zou bovendien extra css nodig hebben om columns zichtbaar te maken (zoals op site Bootstrap) en dat willen we niet.